admin-docs/creating-entity-pages.md

```diff
@@ -0,0 +1,185 @@
+# CREATING ENTITY PAGES
```

Comment:
Suggested entry for allowing users to check their datasets:

## Viewing your datasets

You can check an entities’ datasets by following this procedure.

1. Create a new page with a title and the following contents:
```
<h1>My entity Datasets</h1>
<h3>Formatted entity data</h3>
<pre id="formatted-entity"></pre>
<hr />
<h3>Raw entity data</h3>
<pre id="raw-entity"></pre>
<hr />
<h3>Tamplate data</h3>
<pre id="template-data"></pre>
```
2. Add the following script to the page:
```
document.getElementById('formatted-entity').innerHTML = JSON.stringify(datasets.entity, undefined, 1);
document.getElementById('raw-entity').innerHTML = JSON.stringify(window.store.getState().page.datasets.toJS().entityRaw, undefined, 1);
document.getElementById('template-data').innerHTML = JSON.stringify(window.store.getState().page.datasets.toJS().template, undefined, 1);
```
3. Enable the page to be used as an entity view.
4. Assign the page to any template you would like for viewing it's entities datasets.

Now, when you go to `view` any of the entities using that template, you will be able to see all the datasets available.

Reply:
Great! Added.